conf/uppmax-devel.config

```diff
@@ -81,7 +81,10 @@ process {
     memory = { 16.GB * task.attempt }
   }
   $featureCounts {
-    module = ['bioinfo-tools', 'subread/1.5.1']
+    module = ['bioinfo-tools', 'subread/1.5.1'] 
+  }
+  $merge_featureCounts{
+    module =['bioinfo-tools'] 
```

Comment:
No need for this - you only need `bioinfo-tools` if you're then going to load a second module which is part of that.

You can just skip this entire block here as we don't need to specify anything outside of the defaults.